src/visualBuilder/components/FieldLocationAppList.tsx

```diff
@@ -30,6 +31,9 @@ interface FieldLocationAppListProps {
     apps: App[];
     position: "left" | "right";
     toolbarRef: React.RefObject<HTMLDivElement>;
+    domEditStack:CslpData[]
```

Comment:
This should already be with the SDK? Since the SDK itself send this in the first place?